easybuild/easyconfigs/b/Boost/Boost-1.70.0-CrayGNU-19.10-python2.eb

```
@@ -45,7 +40,7 @@ dependencies = [
 # for python3 the corresponding lib is libboost_python36.so
 sanity_check_paths = {
     'files': ['lib/libboost_system.so', 'lib/libboost_mpi.so',
-              'lib/libboost_python%s%s.so' % (local_py_maj_ver, local_py_min_ver)],
+              'lib/libboost_python%%(pymajver)s%s.so' % _pyminver],
```

Thanks @ekouts: is it possible to use the same format for both `major` and `minor` versions, as in my suggestion ? 
```suggestion
              'lib/libboost_python%s%s.so' % (pymajver, _pyminver)],
```

---

`pymajver` is a template variable, while `_pyminver` is a local variable. The problem is template variables don't work with this syntax.

---

Ok, never mind!